app/services/search_remote_jobs_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class SearchRemoteJobsService
+  include JobDesconstruction
+
+  def initialize(params)
+    @link = params.dig('link')
+    @main_link = params.dig('main_link')
+    @link_search = params.dig('link_search')
+    @card_group_search = params.dig('card_group')
+    @title_search = params.dig('title_search')
+    @amount_search = params.dig('amount_search')
+    @name = params.dig('name')
+  end
+
+  def execute
+    cards.map { |card| insert_job(card) }
+  end
+
+  private
+
+  def cards
+    page.css(@card_group_search)
+  end
+
+  def page
+    Mechanize.new.get(@link)
+  end
+
+  def insert_job(card)
+    @card = card
+    Job.create(
```

Comment:
```suggestion
    Job.create!(
```
Sempre melhor utilizar com o `!` porque a exception não é levantada assim, desta forma nunca iriamos saber que aqui poderia ter um bug sem debugar a fundo